### PR TITLE
Rest controllers

### PR DESCRIPTION
**Pull request by SergiiKravchuk**
Adding controller functionality.

### CODE REVIEW

**Comment by SergiiKravchuk** [comment]
Also need to add a gitignore file with excluding '.idea' directory.

File: src/main/java/com/onseo/courses/ds/controllers/TestClassResponse.java [hunk 1]
```diff
@@ -0,0 +1,15 @@
+package com.onseo.courses.ds.controllers;
+
+//class for tmp controller testing
+public class TestClassResponse {
```
Comment (MaksymChervinskyi) [inline src/main/java/com/onseo/courses/ds/controllers/TestClassResponse.java]: I don't see any test here
Please move this class to "test" directory

File: src/main/java/com/onseo/courses/ds/controllers/QuestionController.java [hunk 2]
```diff
@@ -0,0 +1,35 @@
+package com.onseo.courses.ds.controllers;
+
+import org.springframework.web.bind.annotation.*;
+
+//adds start path for requests
+@RestController()
+@RequestMapping("/api")
+public class QuestionController {
```
Comment (MaksymChervinskyi) [inline src/main/java/com/onseo/courses/ds/controllers/QuestionController.java]: Split to Interface and implementation

File: src/main/java/com/onseo/courses/ds/controllers/UserController.java [hunk 3]
```diff
@@ -0,0 +1,37 @@
+package com.onseo.courses.ds.controllers;
+
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.PostMapping;
+import org.springframework.web.bind.annotation.RequestParam;
+import org.springframework.web.bind.annotation.RestController;
+
+@RestController
+public class UserController {
```
Comment (MaksymChervinskyi) [inline src/main/java/com/onseo/courses/ds/controllers/UserController.java]: Split to Interface and implementation

File: src/main/java/com/onseo/courses/ds/interfaces/BaseQuestionController.java [hunk 4]
```diff
@@ -0,0 +1,26 @@
...
+public interface BaseQuestionController {
+
+    @GetMapping("/questions")
+    public String getQuestions();
+
+    @PostMapping("/answers")
+    public String getAnswers(@RequestParam(name = "usesId") Integer user_id,
+                             @RequestParam(name = "questionId") Integer question_id,
+                             @RequestParam(name = "answerIds") Integer[] answers_id
+    );
```
Comment (MaksymChervinskyi) [inline src/main/java/com/onseo/courses/ds/interfaces/BaseQuestionController.java]: parameters mist be "String"

**Pull request merged**